machine/mellanox/mlnx_x86/i2ctools/series

```diff
@@ -1,4 +1,2 @@
-0001-Increasing-error-flow-verbosity-eeprom_read_byte.patch
-0002-fixing-DEBUG-compilation.patch
-0003-Adding-Mellanox-platform-support.patch
-0004-write_sys_eeprom-taking-eeprom-open-out-of-the-eepro.patch
+0001-Mellanox-i2c-platform-support.patch
+0002-Mellanox-i2c-eeprom_force.patch
```

Comment:
Why do you need to patch i2ctools at all?  Now you are using the busybox implementation of `onie-syseeprom` I would expect all the i2ctools patches to go away.

Reply:
I made just in case support also for onie-syseeprom tool and tested it.
I'll remove completely unnecessary i2ctools directory under Mellanox machine.